tests/Unit/NumericalAlgorithms/LinearSolver/ConjugateGradient/Test_ResidualMonitorActions.cpp

```diff
@@ -179,7 +179,7 @@ struct Metavariables {
                                     MockObserverWriter<Metavariables>>;
   using system = System;
   using const_global_cache_tag_list = tmpl::list<>;
-  enum class Phase { Initialize, Exit };
+  enum class Phase {};
```

Comment:
Do you need phase at all? If so, why in this test and not the other one (gmres)?

Reply:
It's needed because the full `ResidualMonitor` template is instantiated to retrieve its `const_global_cache_tag_list`(https://github.com/sxs-collaboration/spectre/pull/1483/files#diff-9052f4d991c95326d8081a6b651ed9a1R83). Both tests do that.